src/library/scala/PartialFunction.scala

```diff
@@ -159,6 +165,16 @@ trait PartialFunction[-A, +B] extends (A => B) { self =>
  *  @since   2.8
  */
 object PartialFunction {
+
+  final class ElementWiseExtractor[-A, +B] private[PartialFunction] (private val pf: PartialFunction[A, B]) extends AnyVal {
+    def unapplySeq(a: Seq[A]): Option[Seq[B]] = {
```

Comment:
seq not a

Reply:
Renamed in https://github.com/scala/scala/pull/7111/commits/d7051d5c2409016d0ef376db97bd1ed7c3552b47